modules/claims_api/config/schemas/veteran_identifier.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Veteran Identifier",
+  "type": "object",
+  "required": [
+    "ssn",
+    "firstName",
+    "lastName",
+    "birthdate"
+  ],
+  "properties": {
+    "ssn": {
+      "type": "string"
```

Comment:
should we have some validation on the string submitted here?
https://github.com/department-of-veterans-affairs/vets-api/blob/3d694a147d970392927066b244eae42135ec334d/modules/claims_api/app/controllers/claims_api/v2/veterans.rb#L21 is an example of what we could require for ssn

Reply:
[Added here](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/13e7dab71e68ae8ea80874ac5158c72df10f2997).